tests/testthat/test-github_document.R

```diff
@@ -0,0 +1,26 @@
+
+test_that("toc has correct identifier", {
+  local_edition(3)
+  skip_on_cran() # avoid pandoc issue on CRAN
+  tmp_file <- withr::local_tempfile()
```

Comment:
Rather than the `if` statement below, I'd recommend `skip_if_not_pandoc("2.10.1")` (creating the function as necessary); that way the test will be recorded as being skipped so that you know it hasn't run.

Reply:
Oh indeed. At first I wanted to make two snapshots : one for Pandoc 2.10.1+ and one for earlier version. 
This would require a `if` or two `test_that()` with different `skip_*`
However it is currently not easy to generate snapshot for different pandoc versions.